index.html

```diff
@@ -21,22 +19,8 @@
 			},
 		],
 		"shortName": "tipjar",
-		"specStatus": "CG-DRAFT",
+		"specStatus": "unofficial",
 		"subjectPrefix": "[tipjar]",
```

Comment:
You probably don't want this either, as we don't generally use mailing lists anymore.